Update DO how-to v0.18.1 (#710)
meilisearch/documentation: resources/howtos/digitalocean_droplet.md

## Configure production settings in your MeiliSearch Droplet

Configuring your MeiliSearch in a PRODUCTION environment on DigitalOcean droplet is very straightforward. Establish an SSH connection with your droplet and a script will guide you through the process.

### 1. Make your domain name point to your droplet

If you want to use your own domain name (or sub-domain), add `A record` in your domain name provider account.

![Domain to  MeiliSearch](/digitalocean/11.domain-a-record.png)

This should work out of the box. Your domain name should be now linked to your MeiliSearch instance. You can now do a health check to verify that your instance is running and your DNS is well configured:

```bash
$ curl -v http://<your-domain-name>/health
```

The server should answer with a `204 No content` status code as shown in the example below:

```bash
...
< HTTP/1.1 204 No Content
...
```

![Domain to  MeiliSearch](/digitalocean/11.working-domain.png)

### 2. Set API KEY and SSL (HTTPS)

> Meilisearch is running in a DEVELOPMENT environment. You haven't set up an API KEY (anyone can read/write from your MeiliSearch) and you aren't using HTTPS yet, which makes this configuration uinsafe to une in PRODUCTION. To start the configuration process, connect via SSH to your new MeiliSearch Droplet and follow the instructions:

Reviewer: ```suggestion
Meilisearch is running in a DEVELOPMENT environment. You haven't set up an API KEY (anyone can read/write from your MeiliSearch) and you aren't using HTTPS yet, which makes this configuration unsafe to use in PRODUCTION.
To start the configuration process, connect via SSH to your new MeiliSearch Droplet and follow the instructions:
```

### 2.1. Run the configuration script

Open a terminal and start a new SSH connection with the IP you got from DigitalOcean.

Write in your terminal `ssh root@<your-ip-address>` and press Enter to establish connection:

```bash

> ssh meilisearch@42.42.42.42

Reviewer: ```suggestion
$ ssh meilisearch@42.42.42.42
```

```

Write `yes` and press Enter to accept the authentication process.

A script will run automatically, asking for your settings and desired configuration. If you want to run this script again anytime, you can do so by using the following command:

```bash

> meilisearch-setup

Reviewer: ```suggestion
$ meilisearch-setup
```
